bigtable/google/cloud/bigtable/row.py

```diff
@@ -86,7 +85,7 @@ class _SetDeleteRow(Row):
     * :meth:`delete_cell`
     * :meth:`delete_cells`
 
-    :type row_key: bytes
+    :type row_key: str
```

Comment:
What is the rationale here?  The [backend docs](https://cloud.google.com/bigtable/docs/reference/data/rpc/google.bigtable.v2#google.bigtable.v2.Row) explicitly state that row keys are bytes, not text.